src/main/java/com/google/flourbot/entity/EntityModule.java

```diff
@@ -0,0 +1,86 @@
+package com.google.flourbot.entity;
+
+import com.google.flourbot.entity.EntityInterface;
+import com.google.flourbot.datastorage.FirebaseDataStorage;
+import com.google.cloud.firestore.QueryDocumentSnapshot;
+import com.google.flourbot.entity.action.Action;
+import com.google.flourbot.entity.action.SheetAppendAction;
+import com.google.flourbot.entity.trigger.Trigger;
+import com.google.flourbot.entity.trigger.CommandTrigger;
+import java.util.Map;
+
+import java.lang.Exception;
+
```

Comment:
EntityModuleImpl is the default naming scheme if you can find something specific like adding Firebase in case of the DataStorage module.

Reply:
Ah okay, so fooModule is the name for interface and fooModuleImpl is the naming for the actual class